onnxruntime/core/framework/session_options.h

```diff
@@ -64,6 +64,10 @@ struct SessionOptions {
   // See class 'OrtValuePatternPlanner'.
   bool enable_mem_pattern = true;
 
+  bool enable_mem_reuse = true;
+
+  bool transfer_ownership_intermediate_output_tensors = false;
```

Comment:
>transfer_ownership_intermediate_output_tensors [](start = 7, length = 46)

This requires more comments

Reply:
Yes, sir. I have added comments and will push in the next commit.

---
In reply to: [601704327](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r601704327) [](ancestors = 601704327)